docs/DOCKER.md

```diff
@@ -41,52 +41,22 @@ curl https://raw.githubusercontent.com/elastic/connectors-python/main/config.yml
 If you're running the Connector Service against a dockerised version of Elasticsearch and Kibana, your config file will look like this:
 
 ```
-elasticsearch:
-  host: http://host.docker.internal:9200
-  username: elastic
-  password: <YOUR_PASSWORD>
-  ssl: true
-  bulk:
-    queue_max_size: 1024
-    queue_max_mem_size: 25
-    display_every: 100
-    chunk_size: 1000
-    max_concurrency: 5
-    chunk_max_mem_size: 5
-    concurrent_downloads: 10
-  request_timeout: 120
-  max_wait_duration: 120
-  initial_backoff_duration: 1
-  backoff_multiplier: 2
-  log_level: info
-
-service:
-  idling: 30
-  heartbeat: 300
-  max_errors: 20
-  max_errors_span: 600
-  max_concurrent_syncs: 1
-  job_cleanup_interval: 300
-  log_level: INFO
-  
-# connector information
-connector_id: <CONNECTOR_ID_FROM_KIBANA>
-service_type: <DESIRED_SERVICE_TYPE>
-
-sources:
-  mongodb: connectors.sources.mongo:MongoDataSource
-  s3: connectors.sources.s3:S3DataSource
-  dir: connectors.sources.directory:DirectoryDataSource
-  mysql: connectors.sources.mysql:MySqlDataSource
-  network_drive: connectors.sources.network_drive:NASDataSource
-  google_cloud_storage: connectors.sources.google_cloud_storage:GoogleCloudStorageDataSource
-  azure_blob_storage: connectors.sources.azure_blob_storage:AzureBlobStorageDataSource
-  postgresql: connectors.sources.postgresql:PostgreSQLDataSource
-  oracle: connectors.sources.oracle:OracleDataSource
-  mssql: connectors.sources.mssql:MSSQLDataSource
+# When connecting to your cloud deployment you should edit the host value
+elasticsearch.host: http://host.docker.internal:9200
+
+# Using `api_key` is recommended instead of `username`/`password`
+elasticsearch.api_key: <ELASTICSEARCH_API_KEY>
+# elasticsearch.username: elastic
+# elasticsearch.password: <YOUR_PASSWORD>
+
+connectors:
```

Comment:
`sidebar`: should these be commented out in the _actual_ config file. Wondering if that's unnecessary 🤔

Reply:
They are commented out in the actual config file. Let's get rid of them. I added a note at the bottom of the config that you can use username/password as well.